src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -266,10 +302,14 @@ public IEnumerable<FrameworkSpecificMapping> EquivalentProfiles
                     _subSetFrameworks = new KeyValuePair<string, string>[]
                         {
                             // .NET is a subset of DNX
-                            new KeyValuePair<string, string>(FrameworkConstants.FrameworkIdentifiers.Net, FrameworkConstants.FrameworkIdentifiers.Dnx),
-
-                            // DotNet is a subset of DNXCore
-                            new KeyValuePair<string, string>(FrameworkConstants.FrameworkIdentifiers.NetPlatform, FrameworkConstants.FrameworkIdentifiers.DnxCore),
+                            new KeyValuePair<string, string>(
+                                FrameworkConstants.FrameworkIdentifiers.Net,
+                                FrameworkConstants.FrameworkIdentifiers.Dnx),
+
+                            // NetStandard is a subset of DNXCore
+                            new KeyValuePair<string, string>(
```

Comment:
This mapping is incorrect. It would say that netstandard5.5 is compatible with dnxcore5.5, basically every netstandard version would install into dnxcore. 

The dnxcore -> dotnet mapping needs to stay in place for backwards compatibility. Just add the old one back and leave netstandard out for this.